homeassistant/components/person/__init__.py

```diff
@@ -318,7 +350,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     )
     await storage_collection.async_load()
 
-    hass.data[DOMAIN] = (yaml_collection, storage_collection)
+    hass.data[DOMAIN] = (yaml_collection, storage_collection, entity_component)
```

Comment:
Migration path for existing data stored in this key?

Reply:
It's only used in this file. Other integrations should never use the private data stored in `hass.data`.

Reply:
Roger.